routes.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const fp = require('fastify-plugin')
+const readFileSync = require('fs').readFileSync
+const resolve = require('path').resolve
+
+const files = {
+  html: readFileSync(resolve('static', 'index.html'), 'utf8'),
```

Comment:
Truly this may wrong. `resolve` resolves the path from your `cwd` (the application path)

Maybe you can use this https://github.com/fastify/fastify-static